index.html

```diff
@@ -8277,22 +8277,134 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
 </section>
 
 <section class="informative changed"><h3>Included Nodes</h3>
-  <p>Sometimes it is useful to list node objects as part of another node object.
-    For instance, to represent a set of resources which are used by some other
-    resource.
-    In JSON-LD, these resources could all be contained as members of an array,
-    but this does not conveniently allow them to share a common context.</p>
+  <p>
+    Most of the examples so far were about a top level node with its properties referring to, possibly, other nodes in a tree-like fashion.
+    However, there are cases when the data should combine <em>several</em> such objects within one JSON-LD document.
+    In JSON-LD, these resources could all be contained as members of an array:
+  </p>
 
-  <p>Included resources are described in
-    <a data-cite="JSON.API#fetching-includes">Inclusion of Related Resources</a> of [[[JSON.API]]] [[JSON.API]]
-    as a way to include related resources associated with some primary resource.</p>
+  <pre class="example input" id="bush-in-array" title="Simple data with several top level nodes" data-transform="updateExample">
+  <!--
+[
+    {
+        "@id": "http://manu.sporny.org/about#manu",
+        "@type": "http://xmlns.com/foaf/0.1/Person",
+        "name": "Manu Sporny",
+        "knows": { "@id": "https://greggkellogg.net/foaf#me" }
+    }, {
+        "@id": "https://greggkellogg.net/foaf#me",
+        "@type": "http://xmlns.com/foaf/0.1/Person",
+        "name": "Gregg Kellogg",
+        "knows": { "@id": "http://manu.sporny.org/about#manu"}
+    }  
+]
+  -->    
+  </pre>
 
-  <p>In JSON-LD, <a>included blocks</a> may be used to collect such secondary <a>node objects</a>
-    which can be referenced from a primary <a>node object</a>.
-    Semantically, this is the same as if the node objects were embedded, or were contained
-    in some enclosing <a>array</a>.</p>
+  <p>
+    The problem with this approach is that we lose the ability to use a common context.
```

Comment:
Is this true? This seems like the use case for `{"@context": {}, "@graph": [ ... ]}` ?

Reply:
e.g.:  http://tinyurl.com/y54yasoh

Reply:
Well, yes, the solution so far to the problem is to use `@graph`. But it turns out that `@included` can play exactly the same role, except that using that keyword (instead of `@graph`) is cleaner. The problem with using `@graph` for something like that is that it looks like we are talking about a dataset when, in fact, we are not!

B.t.w., and that is relevant, the example is almost exactly the same is a few chapter below (example 111 in the PR) where datasets/named graphs are described. But there is difference between named graphs (or data sets) and a single graph which can then be handled via `@included`.

B.t.w., and I forgot to do that: if we agree with this PR, then the text around 111 should be redone. Of course, 111 remains valid due to backward compatibility reasons but I would propose that the `@included` should be promoted instead.